packages/ui/src/ui-component/extended/SpeechToText.jsx

```diff
@@ -76,6 +86,60 @@ const speechToTextProviders = {
                 credentialNames: ['assemblyAIApi']
             }
         ]
+    },
+    [SpeechToTextType.LOCALAI_STT]: {
+        label: 'LocalAi STT',
+        name: SpeechToTextType.LOCALAI_STT,
+        icon: localAISVG,
+        url: 'https://localai.io/features/audio-to-text/',
+        inputs: [
+            {
+                label: 'Connect Credential',
+                name: 'credential',
+                type: 'credential',
+                credentialNames: ['localAIApi']
+            },
+            {
+                label: 'Base URL',
+                name: 'baseUrl',
+                type: 'string',
+                description: 'The base URL of the local AI server'
+            },
+            {
+                label: 'Language',
+                name: 'language',
+                type: 'string',
+                description:
+                    'The language of the input audio. Supplying the input language in ISO-639-1 format will improve accuracy and latency.',
+                placeholder: 'en',
+                optional: true
+            },
+            {
+                label: 'Model',
+                name: 'model',
+                type: 'string',
+                rows: 4,
+                description: `The model to load. `,
+                placeholder: 'whisper-1',
```

Comment:
we can have the `default : whisper-1` as well if thats the default model that is going to be used by LocalAI if nothing if specified. Also to be clearer, we can add description saying if not specified, which model will be used by default. We should get rid of `rows: 4`